src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -62,6 +66,13 @@ export default function useBulkImportFields() {
       );
       return bulkSightingFields.map(f => {
         const additionalProperties = {};
+        if (
+          f?.fieldType === 'select' ||
+          f?.fieldType === 'multiselect'
+        ) {
+          additionalProperties.type = 'select';
```

Comment:
select, checkbox, and string seem to be FlatFile-specific designations. I need to learn about whether they even accommodate multi-select in this way, and, if not, what we can do about it (treat it as a string and have some way to custom validate it?? Seems like a lot.)